index.html

```diff
@@ -349,7 +349,7 @@
             .grid {
                 display: grid;
                 grid-template-columns: repeat(auto-fill, minmax(9rem, 1fr));
-                grid-auto-rows: 6rem;
+                grid-auto-rows: 6.25rem;
```

Comment:
I think `grid-auto-rows: min-content` would fit better (since it automatically adjusts to the content height)